client/my-sites/stats/stats-comment-followers-page/index.jsx

```diff
@@ -138,23 +97,12 @@ class StatModuleFollowersPage extends Component {
 			labelLegend = translate( 'Follower' );
 			valueLegend = translate( 'Since' );
 		}
-
-		const emailExportUrl = followType === 'email'
-			? 'https://dashboard.wordpress.com/wp-admin/index.php?page=stats&blog=' + siteId + '&blog_subscribers=csv&type=email'
-			: null;
-
 		return (
 			<div className="followers">
-				<QuerySiteStats statType={ statsType } siteId={ siteId } query={ query } />
-				<SectionHeader label={ translate( 'Followers' ) }>
-					{ emailExportUrl
-						? ( <Button compact href={ emailExportUrl }>{ translate( 'Download Data as CSV' ) }</Button> )
-						: null }
-				</SectionHeader>
+				<QuerySiteStats statType="statsCommentFollowers" siteId={ siteId } query={ query } />
+				<SectionHeader label={ translate( 'Comments Followers' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Comment followers' )` *ES Score: 14.56*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/stats/use-people-for-followers)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).